web/components/NavBlock/NavBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import Link from 'next/link';
+import clsx from 'clsx';
+import styles from './NavBlock.module.css';
+
+export const NavBlock = () => (
+  <nav className={styles.nav}>
+    <ul className={styles.list}>
+      <li className={styles.item}>
+        <Link href="/program">
+          <a className={styles.link}>Program</a>
+        </Link>
+      </li>
+      <li
+        className={clsx(
+          styles.fakeItem,
+          styles.mobile,
+          styles.tablet,
+          styles.desktop
+        )}
+        aria-hidden="true"
+      ></li>
+      <li
+        className={clsx(styles.fakeItem, styles.tablet, styles.desktop)}
+        aria-hidden="true"
+      ></li>
```

Comment:
Nit: In JSX, empty XML nodes should be collapsed.<sup>[citation needed]</sup> 
My editor gets a little upset otherwise. But this is personal (jsx) preference.

Reply:
Thanks, JSX is kind of an "uncanny valley" :) 249a1f591e29f1d85a260039031ca0cdf7540a46